nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -189,6 +195,13 @@ void RegulatedPurePursuitController::configure(
     plugin_name_ + ".extended_collision_check_path_end_leniency",
     extended_collision_check_path_end_leniency_);
 
+  node->get_parameter(
+    plugin_name_ + ".rotate_to_heading_min_angular_vel",
+    rotate_to_heading_min_angular_vel_);
+  node->get_parameter(
+    plugin_name_ + ".rotate_to_heading_proportional_gain",
+    rotate_to_heading_proportional_gain_);
```

Comment:
Not necessarily has to be within 0 and 1

Reply:
It must be a positive number though.. that i can check. 
Would would i do if its not? I could throw an exception.
I wouldn't default it to a valid number silently because wrong parameterization will go unnoticed

Reply:
To be honest I cannot think of a case here where you'd use P > 1. Can you?

Reply:
Actually there are other parameter validations in the constructor where it defaults and prints a warning..
I would rather throw an exception for this one, but will make this validation inconsistent..
So I'll just default

Reply:
Its a proportional term basically PID controller (without I or D in our case).
It can even be 1000. (angular vel is capped to max of course)
If distance to goal heading is 0.5rads, maybe I want the angular velocity to be 1rad/s. In that case P = `2.0`